core/services/fluxmonitorv2/integrations_test.go

```diff
@@ -432,9 +432,9 @@ func TestFluxMonitor_Deviation(t *testing.T) {
 	type k struct{ latestAnswer, updatedAt string }
 	expectedMeta := map[k]int{}
 
-	reportPrice := int64(100)
+	reportPrice := atomic.NewInt64(100)
```

Comment:
These `reportPrice` vars had a mix of atomic and non-atomic access.

Reply:
ah nice find! yeah that's likely part of what they warn about in the docs on using sync/atomic...